src/components/ShareStructure/ShareStructure.vue

```diff
@@ -1,427 +1,731 @@
-                  <v-radio :value="false">
-                    <template v-slot:label>
-                      <v-row><v-col cols="6">
-                      <v-text-field
-                        filled
-                        label="Maximum Number of Shares"
-                        id="txt-max-shares"
-                        v-model="shareStructure.maxNumberOfShares"
-                        persistent-hint
-                        :hint="'Enter the maximum number of shares in the ' + shareStructure.type"
-                        :rules="getMaximumShareRule()"
-                        :disabled="hasNoMaximumShares"/>
-                    </v-col></v-row>
-                    </template>
-                  </v-radio>
-                  <v-radio :value="true" label="No maximum" id="lbl-no-maximum" v-if="isNoMaxSharesVisible"/>
-                </v-radio-group>
-
-                <v-divider class="separator" />
-
-                <v-radio-group
-                  v-model="hasNoParValue"
-                  column
-                  class="radio-group"
-                  @change="changeParValueFlag()" v-show="isClass">
-                  <v-radio :value="false" id="radio-par-value">
-                    <template v-slot:label>
-                      <v-row>
-                        <v-col cols="6">
-                          <v-text-field
-                            filled
-                            label="Par Value"
-                            id="class-par-value"
-                            v-model="shareStructure.parValue"
-                            :rules="getParValueRule()"
-                            hint="Enter the initial value of each share"
-                            persistent-hint/>
-                        </v-col>
-                        <v-col cols="6">
-                          <v-select
-                            :items="getCurrencyList()"
-                            filled
-                            label="Currency"
-                            v-model="shareStructure.currency"
-                            :rules="getCurrencyRule()"
-                            item-text="`${data.item.name}, ${data.item.code}`"
-                            item-value="code"
-                            id='class-currency'>
-                            <template slot="selection" slot-scope="data">
-                               {{ data.item.name }} ({{ data.item.code }})
-                            </template>
-                            <template slot="item" slot-scope="data">
-                              {{ data.item.name }} ({{ data.item.code }})
-                            </template>
-                          </v-select>
-                        </v-col>
-                      </v-row>
+  <v-card flat id="share-structure">
+
+    <!-- Summary Section -->
+    <div id="share-summary">
+      <!-- Summary Header -->
+      <div class="share-summary-header" >
+        <v-icon>mdi-file-tree</v-icon>
```

Comment:
Header icon colour should be #38598A.
Please fix any other instances that you find.

Reply:
done and done